src/applications/deploying-apps.md

```diff
@@ -61,10 +61,31 @@ It is possible to configure a few of the [globally tunable](../applications/mana
 Setting                                         | Description
 ----------------------------------------------- | ---------------------------------
 DEIS_DEPLOY_BATCHES                             | the number of pods to bring up and take down sequentially during a scale (default: number of available nodes)
+DEIS_DEPLOY_TIMEOUT                             | deploy timeout in seconds - there are 2 deploy methods, current (RC) and Deployments (see below) and this setting affects those a bit differently (default: 120)
 DEIS_KUBERNETES_DEPLOYMENTS                     | if enabled [Deployments][] is used to handle an application deploy instead of [ReplicationControllers][]
                                                 | any value is acceptable to turn on [Deployments][], to turn it off either remove or pass an empty string (default: off)
 KUBERNETES_DEPLOYMENTS_REVISION_HISTORY_LIMIT   | how many [revisions][[kubernetes-deployment-revision]] Kubernetes keeps around of a given Deployment (default: all revisions)
 
+### Deploy Timeout
+
+Deploy timeout in seconds - There are 2 deploy methods, current (RC) and Deployments (see below) and this setting affects those a bit differently.
+
+#### RC deploy
+
+The timeout is the base timeout which is used per `DEIS_DEPLOY_BATCHES`
```

Comment:
thoughts on the below? seems like it would be more clear and less repetitive:

This deploy timeout defines how long to wait for each batch to complete in `DEIS_DEPLOY_BATCHES`